todo_app/templates/index.html

```diff
@@ -4,15 +4,83 @@
 {% block content %}
   <div class="jumbotron">
     <h1 class="display-4">To-Do App</h1>
-    <p class="lead">Just another to-do app.</p>
+    <p class="lead">This is not just any to-do app...</p>
   </div>
 
   <div class="row">
     <div class="col-md-12">
       <h2>Items</h2>
-      <ul class="list-group mb-4">
+      <ul id="todo_list" class="list-group mb-4">
         <!-- List the todo items here -->
+        {% for item in items %}
+        <li id="li{{ item.id}}"><span>{{ item.title }} - </span><span id="status{{ item.id}})" onclick="updateStatus({{ item.id }}, event)">{{ item.status }}</span> <span onclick="deleteItem({{ item.id }})">(Delete)</span></li>
```

Comment:
Splitting these spans out onto separate lines would help to make this quite a bit easier to parse

Reply:
Wanted to try and represent this as a line, but given HTML doesn't care agree its a bit smoother so changed.